dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/HostConnectionPoolImpl.java

```diff
@@ -304,7 +308,7 @@ private ConnectionPoolActive(HostConnectionPoolImpl<CL> cp) {
                     }
 				}
 				monitor.incConnectionCreateFailed(host, e);
-				throw new DynoConnectException(e);
+				throw new DynoConnectException(e); // TODO - FatalConnectionException ?
```

Comment:
Good catch - I was thinking through solutions and put that as a marker for one of them. I forgot to remove it.